changelogs/fragments/84574-target_log_severity.yml

```
@@ -0,0 +1,2 @@
+breaking_changes:
+  - Ansible's systemd logs now contain a PRIORITY field (deault value: 5).  Downstream programs that inspect Ansible's systemd logs should be updated if they expect the PRIORITY field to be missing (https://github.com/ansible/ansible/pull/84574).
```

https://dev.azure.com/ansible/ansible/_build/results?buildId=135086&view=logs&j=6b8fca62-2c63-5c74-2051-15ea31c8c104&t=e6a77036-d6ac-54d6-bbbb-976ea775a86e&l=479

It's a bit unobvious, but since you have that `: ` in the middle (with a whitespace), the YAML parser makes it a key-value mapping. Here's one way to fix it:
```suggestion
  - >-
    Ansible's systemd logs now contain a PRIORITY field (deault value: 5).
    Downstream programs that inspect Ansible's systemd logs should be updated if
    they expect the PRIORITY field to be missing
    (https://github.com/ansible/ansible/pull/84574).
```